pollination/honeybee_vtk/export_images.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+from pydantic.typing import IntStr
+
+
+@dataclass
+class ExportImages(Function):
+    """Export images from am hbjson file."""
+
+    hbjson_file = Inputs.file(
+        description='HBJSON file.',
+        path='input.hbjson'
+    )
+
+    name = Inputs.str(
+        description='Name of image files.',
+        default='Camera'
```

Comment:
I really think we should work on this and make the default name to be based on the view names.

If the view object has no identifier then we should create one for it. cc: @chriswmackey

Reply:
A view object does have an identifier property. This input here is for hbjsons that don't have radiance views. Now that we are talking about it, I think that I can remove this input also and use a default identifier under the hood.

Reply:
I see. This means that you will be exporting the default views that are built in. Right? If that's the case we should just name them as Top, Right, Left, etc. We don't need the name camera before them.

Reply:
I have captured this in an [issue](https://github.com/ladybug-tools/honeybee-vtk/issues/193) to do it later.